src/main/java/org/cryptomator/frontend/dokany/MountFactory.java

```diff
@@ -103,11 +103,11 @@ public Mount mount(Path fileSystemRoot, Path mountPoint, String volumeName, Stri
 	 * @param volumeName The name of the drive as shown to the user.
 	 * @param fileSystemName The technical file system name shown in the drive properties window.
 	 * @param additionalOptions Additional options for the mount. For any unset option a default value is used. See {@link MountUtil} for details.
-	 * @param afterUnmountAction action to be performed after the filesystem is unmounted.
+	 * @param afterMountExitAction action to be performed after the dokan mount exited (regularly or irregularly)
 	 * @return The mount object.
 	 * @throws DokanyMountFailedException if the mount process is aborted due to errors
 	 */
-	public Mount mount(Path fileSystemRoot, Path mountPoint, String volumeName, String fileSystemName, String additionalOptions, Runnable afterUnmountAction) throws DokanyMountFailedException {
+	public Mount mount(Path fileSystemRoot, Path mountPoint, String volumeName, String fileSystemName, String additionalOptions, Runnable afterMountExitAction) throws DokanyMountFailedException {
```

Comment:
I'm still not 100% happy with the name `afterMountExitAction`. The fact that it contains "mount" might still be misleading. Maybe the shorter `onExit` is generic enough to not create expectations about the timing, so you're forced to rtfm.

Reply:
I renamed it to `onDokanExit` in 8045576. It is more specifc then "onExit", but does not contain anything about the mounting in its name.